src/drawer.js

```diff
@@ -383,6 +383,7 @@ $.Drawer.prototype = /** @lends OpenSeadragon.Drawer.prototype */{
 
         this.context.save();
         this.context.globalAlpha = opacity;
+        this.context.globalCompositeOperation = compositeOperation;
```

Comment:
It is set to default in openseadragon.js

1073             // APPEARANCE
1074             opacity:                    1,
1075             compositeOperation:         'source-over',
1076             placeholderFillStyle:       null,


----- Original Message -----
From: "Ian Gilman" <notifications@github.com>
To: "openseadragon/openseadragon" <openseadragon@noreply.github.com>
Cc: "meihuisu" <mei@isi.edu>
Sent: Friday, January 15, 2016 9:16:07 AM
Subject: Re: [openseadragon] adding setting of globalCompositeOperation for tiledImage (#814)

> @@ -383,6 +383,7 @@ $.Drawer.prototype = /** @lends OpenSeadragon.Drawer.prototype */{
>  
>          this.context.save();
>          this.context.globalAlpha = opacity;
> +        this.context.globalCompositeOperation = compositeOperation;

compositeOperation is optional, so we should only set it to the context if it exists. 

Also, please add compositeOperation to the doc comment here. 

---
Reply to this email directly or view it on GitHub:
https://github.com/openseadragon/openseadragon/pull/814/files#r49878837